x-pack/plugins/security_solution/public/entity_analytics/components/entity_details_flyout/tabs/risk_inputs/risk_inputs_tab.tsx

```diff
@@ -172,6 +160,16 @@ export const RiskInputsTab = ({ entityType, entityName }: RiskInputsTabProps) =>
     );
   }
 
+  const totals = !riskScore
```

Comment:
if `totals` and `displayed` are only used by `AlertInputMessage` maybe we should move this logic into that component?

Reply:
and pass `riskScore` as props? yeah, probably good idea.

Reply:
I think its a matter of opinion, it makes the props less simple, but having this logic closer to where it is used is worth it I think

Reply:
i know, back in pre hooks days we used `recompose` to pipe a bunch of prop transformations via HoC. That meant we could have simpler props and also a clean component definition, and extract all that boilerplate somewhere else. Doesnt work so well with hooks :shrugh: